Riku/Riku/ChangePlayerResources.cpp

```diff
@@ -0,0 +1,14 @@
+//Kacper Walasek
+#include "ChangePlayerResources.h"
+
+ChangePlayerResources::ChangePlayerResources(int player, int resource, int quantity) : player(player), resource(resource), quantity(quantity) {}
+
+std::shared_ptr<Patch> ChangePlayerResources::apply(const GameState& state) const
```

Comment:
Może to tylko ja, ale nazwa apply sugeruje mi, że funkcja modyfikuje gameState, a nie zwraca łatke do zaaplikowania

Reply:
Racja. Ta nazwa jest taka bo wcześniej ta funkcja zmieniała gamestate. Poprawie to

Reply:
Poprawiłem. Jeśli wszystko ok to możesz mergować